include/exiv2/futils.hpp

```diff
@@ -26,8 +26,7 @@
   @date    12-Dec-03, ahu: created<BR>
            02-Apr-05, ahu: moved to Exiv2 namespace
  */
-#ifndef FUTILS_HPP_
-#define FUTILS_HPP_
+#pragma once
```

Comment:
While nearly every compiler supports `#pragma once`, I'd rather keep the old include guards as they are used throughout the whole project and afaik faster with gcc & clang.

Reply:
Ups, I did not notice that the rest of the code is using the guards. I always tend to use `pragma once` and I did this change without thinking about it. I will change it back